### DISCUSSION TITLE
Getting errors with movie script

### DISCUSSION THREAD

**Discussion by wendydunning** [chhoumann/quickadd#570] [Q&A]
Keep getting this error:

 Template Error:
 Failed to load user script Scripts/movies.js.
 Default export is not a function.

Have tried replacing the script with copy of 'raw', changing template (although a screenshot of this same error says "templater error" and I do have templater installed. 

Any help would be greatly appreciated.

**Comment by chhoumann** [comment]
Coming back to this very late - sorry about that.

"Default export is not a function" means QuickAdd found and loaded `Scripts/movies.js`, but the file didn't `module.exports` a function it could run. In practice that's almost always a copy problem: the GitHub HTML page got saved instead of the raw file, only part of the script was copied, or the file was created inside Obsidian and is actually `movies.js.md` under the hood. The "templater error" wording you saw in other screenshots is unrelated - this message comes from QuickAdd's own script loader, not Templater.

The docs page now has a direct download link that serves the raw file: https://quickadd.obsidian.guide/docs/Examples/Macro_MovieAndSeriesScript - grab `movies.js` from there, save it in your vault outside the `.obsidian` folder, and point the macro at it. The script has also picked up several fixes since 2023, so re-downloading is worth it either way.

If it still fails after that, open the developer console (Ctrl+Shift+I), run the macro, and paste the full error text here - that will tell us exactly where it's going wrong.
